### PR TITLE
Edit league season

### PR DESCRIPTION
**Pull request by dogiadat**

### CODE REVIEW

File: app/views/admin/league_seasons/_form.html.erb [hunk 1]
```diff
@@ -16,5 +16,5 @@
     <%= link_to_add_fields t(".add_season_teams"), f, :season_teams %>
   </div>
   <div class="clearfix"></div>
-  <div class="center"><%= f.submit t(".new"), class: "btn btn-primary" %></div>
+  <div class="center"><%= f.submit yield(:submit_button_league_season), class: "btn btn-primary" %></div>
```
Comment (phanbt58) [inline app/views/admin/league_seasons/_form.html.erb]: remove ()

File: app/controllers/admin/league_seasons_controller.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class Admin::LeagueSeasonsController < ApplicationController
-  before_action :load_league, only: [:new, :create, :index]
-  before_action :load_teams, only: [:new, :create]
-  before_action :load_years, only: [:new, :create]
+  before_action :load_league, only: [:new, :create, :index, :edit]
+  before_action :load_teams, only: [:new, :create, :edit, :update]
+  before_action :load_years, only: [:new, :create, :edit, :update]
```
Comment (luuthomvp) [inline app/controllers/admin/league_seasons_controller.rb]: dung except

**Comment by luuthomvp** [comment]
OK

**Comment by phanbt58** [comment]
ok

**Comment by hieuhtk58** [comment]
OK

**Pull request merged**